pom.xml

```diff
@@ -274,6 +274,12 @@
       <version>2.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Why this: to suppress the SLF4J warning message when using AbstractMojoTestCase (and its children); see https://www.slf4j.org/faq.html